evap/evaluation/management/commands/update_course_state.py

```diff
@@ -0,0 +1,14 @@
+import logging
+
+from django.core.management.base import BaseCommand
+
+from evap.evaluation.models import Course
+
+logger = logging.getLogger(__name__)
```

Comment:
whats up with this unused stuff here. i thought you'd use pycharm, didn't you.

;)